src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (x) => {
+    return y => {
```

Comment:
```suggestion
  const getSum = (x) => (y) = >{
```

Можна об'єднати в один рядок, так як  в тебе перша ф-я повертає другу одразу, тому return там не є обов'язковим